packages/ui/package.json

```diff
@@ -44,9 +44,8 @@
     "tailwindcss": "3.2.4"
   },
   "peerDependencies": {
-    "@headlessui/react": "1.7.4",
-    "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "^react": "17.0.2",
+    "^react-dom": "17.0.2"
```

Comment:
Removes headless ui and adds ^ in the package, not the version?

Reply:
good catch!